src/static/css/pages/sub-pages.less

```diff
@@ -16,6 +16,10 @@
     }
 }
 
+.sub-page__grandchild, .sub-page__grandchild + .content_sidebar {
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/01dd47a2574106da6a469140a72c30134746c265